File: app/src/main/java/io/github/romangolovanov/apps/ametro/ui/testing/DebugToast.java
package io.github.romangolovanov.apps.ametro.ui.testing;

import android.content.Context;
import android.widget.Toast;


public class DebugToast {

    public static void show(Context context, String message, int toastLength) {
        Toast.makeText(context, message, toastLength).show();
    }
}

File: app/src/main/java/io/github/romangolovanov/apps/ametro/ui/testing/DebugToast.kt
package io.github.romangolovanov.apps.ametro.ui.testing

import android.content.Context
import android.widget.Toast

object DebugToast {

    fun show(context: Context, message: String, toastLength: Int) {
        Toast.makeText(context, message, toastLength).show()
    }
}
